yarn.lock

```diff
@@ -3199,6 +3199,11 @@
   resolved "https://registry.yarnpkg.com/@vercel/ncc/-/ncc-0.24.0.tgz#a2e8783a185caa99b5d8961a57dfc9665de16296"
   integrity sha512-crqItMcIwCkvdXY/V3/TzrHJQx6nbIaRqE1cOopJhgGX6izvNov40SmD//nS5flfEvdK54YGjwVVq+zG6crjOg==
 
+"@vercel/ncc@0.34.0":
```

Comment:
Lets update all packages?

Reply:
Can do. Just wanted to make sure CLI tests are happy first.

Reply:
2288ec7268d9985f03f4e17672f456eed9f855c0